modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -1,5 +1,27 @@
 [id='con-relnotes-fixed-issues_{context}']
-= Fixed issues
+= Fixed issues in {product} {product-version} and 1.2.2
+
+== Fixed issues in {product} 1.2.2
+
+This section lists fixed issues with {product} 1.2.2:
+
+Dynamic plugins fail to load if the file name in ConfigMap is invalid::
++
+--
+Previously, an invalid dynamic plugin file name in ConfigMap prevented the creation of a Backstage CR. With this release, the Backstage CR can be created even with an invalid plugin file name, and a corresponding log message will be displayed if the {product-short} build fails.
+--
+
+Plugin ID is missing in the permission table in RBAC Administration UI::
++
+--
+Previously, plugin IDs were missing from the permission table in UI accessed from *Administration* -> *RBAC*. With this release, updates to both the RBAC backend and frontend plugins now ensure that plugin IDs are displayed in the table.
+--
+
+Git information for Helm chart-based workflows is no longer shown in the ArgoCD plugin UI::
++
+--
+Previously, the ArgoCD plugin UI displayed Git information for all workflows, including those that were not based on Git. With this release, the ArgoCD plugin UI now only displays Git information for workflows where it is relevant, ensuring that only pertinent information is shown.
```

Comment:
Needlessly verbose description of the situation and its resolution by repeating information that's obvious, ensuring that a longer sentence is used to convey the same information. With this verbesge, you are consistent with the style guide and achieve a larger word count. 

+1 to merge, though I prefer the more terse description I provided.

Reply:
Thank you for the feedback. When not including Jira links, providing context for bug fixes is important. If we add the links, a brief one-liner description of the fix should suffice. We're planning to add Jira links starting with the 1.3 release, so our approach to describing bug fixes may change in future releases.